Starting workspace on openshift 1.4.1

When starting a workspace on openshift 1.4.1 (minishift-1.0.0-beta3), che logs the following error:

```
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://192.168.0.26:8443/api/v1/namespaces/eclipse-che/services. Message: Service "che-ws-i4z9dgl4cpddw1bi" is invalid: [spec.ports[0].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[1].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[2].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[3].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[4].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[5].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[6].protocol: Unsupported value: "tcp": supported values: TCP, UDP]. Received status: Status(apiVersion=v1, code=422, details=StatusDetails(causes=[StatusCause(field=spec.ports[0].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={}), StatusCause(field=spec.ports[1].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={}), StatusCause(field=spec.ports[2].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={}), StatusCause(field=spec.ports[3].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={}), StatusCause(field=spec.ports[4].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={}), StatusCause(field=spec.ports[5].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={}), StatusCause(field=spec.ports[6].protocol, message=Unsupported value: "tcp": supported values: TCP, UDP, reason=FieldValueNotSupported, additionalProperties={})], group=null, kind=Service, name=che-ws-i4z9dgl4cpddw1bi, retryAfterSeconds=nul
l, additionalProperties={}), kind=Status, message=Service "che-ws-i4z9dgl4cpddw1bi" is invalid: [spec.ports[0].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[1].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[2].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[3].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[4].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[5].protocol: Unsupported value: "tcp": supported values: TCP, UDP, spec.ports[6].protocol: Unsupported value: "tcp": supported values: TCP, UDP], metadata=ListMeta(resourceVersion=null, selfLink=null, additionalProperties={}), reason=Invalid, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:315) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.assertResponseCode(OperationSupport.java:268) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:237) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:230) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:208) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:644) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:300) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation$1.apply(BaseOperation.java:317) ~[kubernetes-client-1.4.31.jar:na]
	at io.fabric8.kubernetes.api.model.DoneableService.done(DoneableService.java:26) ~[kubernetes-model-1.0.65.jar:1.0.65]
	at org.eclipse.che.plugin.openshift.client.OpenShiftConnector.createOpenShiftService(OpenShiftConnector.java:480) ~[che-plugin-openshift-client-5.2.0-SNAPSHOT.jar:5.2.0-SNAPSHOT]
	at org.eclipse.che.plugin.openshift.client.OpenShiftConnector.createContainer(OpenShiftConnector.java:176) ~[che-plugin-openshift-client-5.2.0-SNAPSHOT.jar:5.2.0-SNAPSHOT]
	at org.eclipse.che.plugin.docker.machine.MachineProviderImpl.createContainer(MachineProviderImpl.java:529) ~[che-plugin-docker-machine-5.2.0-SNAPSHOT.jar:5.2.0-SNAPSHOT]
	at org.eclipse.che.plugin.docker.machine.MachineProviderImpl.startService(MachineProviderImpl.java:276) ~[che-plugin-docker-machine-5.2.0-SNAPSHOT.jar:5.2.0-SNAPSHOT]
... 13 common frames omitted
```
**OS and version:**    any, openshift 1.4.1
The issue is always reproducible.

--- a/plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/kubernetes/KubernetesService.java
+++ b/plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/kubernetes/KubernetesService.java
@@ -49,7 +49,7 @@ public class KubernetesService {
             int targetPortNumber = portNumber;
             ServicePort servicePort = new ServicePort();
             servicePort.setName(portName);
-            servicePort.setProtocol(protocol);
+            servicePort.setProtocol(protocol.toUpperCase());
             servicePort.setPort(portNumber);
             servicePort.setTargetPort(new IntOrString(targetPortNumber));
             servicePorts.add(servicePort);

